src/aria/utils/Orientation.js

```diff
@@ -33,6 +37,8 @@ module.exports = Aria.classDefinition({
      * Adding a listener while initializing Orientation to listen to the native orientationchange event.
      */
     $constructor : function () {
+        this.$logWarn("This class is deprecated and will be removed from the framework, please use aria.utils.Device instead");
+
```

Comment:
I'd additionally wrap the whole file (i.e. paste this in the first/last line) in sth like 

`/* BACKWARD-COMPATIBILITY-BEGIN GH-1388 */`
...
`/* BACKWARD-COMPATIBILITY-END GH-1388 */`

we usually do that when the whole class is deprecated